plugins/CoreUpdater/Diagnostic/HttpsUpdateCheck.php

```diff
@@ -31,8 +32,14 @@ public function execute()
     {
         $label = $this->translator->translate('Installation_SystemCheckUpdateHttps');
 
-        if (CoreUpdater\Controller::isUpdatingOverHttps()) {
+        if (GeneralConfig::getConfigValue('force_matomo_ssl_request') === 0) {
+            $comment = 'An info message';
```

Comment:
This wasn't meant to be directly committed - needs to be added as a new translation and provide actual info, not the placeholder.